openedx/features/edly/utils.py

```diff
@@ -139,11 +139,8 @@ def update_course_creator_status(request_user, user, set_creator):
     Updates course creator status of a user.
     """
     from course_creators.models import CourseCreator
-    try:
-        course_creator, _ = CourseCreator.objects.get_or_create(user=user)
-        course_creator.state = CourseCreator.GRANTED if set_creator else CourseCreator.DENIED
-        course_creator.note = 'Course creator user was updated by panel admin {}'.format(request_user.username)
-        course_creator.admin = request_user
-        course_creator.save()
-    except CourseCreator.DoesNotExist:
-        LOGGER.info('User %s has no course creator.', user)
+    course_creator, __ = CourseCreator.objects.get_or_create(user=user)
+    course_creator.state = CourseCreator.GRANTED if set_creator else CourseCreator.DENIED
+    course_creator.note = 'Course creator user was updated by panel admin {}'.format(request_user.username)
```

Comment:
`'Course creator user was updated by panel admin "{}"'.format(request_user.email)`

In logs or messages try to user emails instead of usernames.